### ISSUE TITLE
1.21 problem. 

### ISSUE THREAD

**Issue by Deyan2107** [dmulloy2/ProtocolLib#3063]
[17:12:43 INFO]: Deyan21[/ip] logged in with entity id 29 at ([world]-109.94146152794933, 134.58665529011617, 97.31771076883444)
[17:12:43 INFO]: Deyan21 lost connection: Internal Exception: io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:set_objective'
[17:12:43 INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]

![Screenshot_4](https://github.com/dmulloy2/ProtocolLib/assets/148815069/c8283ef8-c392-4973-a0fe-f7a2c1681f3a)

**Comment by Ingrim4** [comment]
Can you please create a protocol dump file using `/protocol dump` command on your server and send us the contents of the created dump file?

**Comment by Deyan2107** [comment]
> Can you please create a protocol dump file using `/protocol dump` command on your server and send us the contents of the created dump file?
[dump-2024-06-29_18.03.12.txt](https://github.com/user-attachments/files/16041077/dump-2024-06-29_18.03.12.txt)

**Comment by Ingrim4** [comment]
Might be [SimpleScore](https://github.com/r4g3baby/SimpleScore) but I'm not sure, can you try to remove it and check if it works again and please provide the full error/log if possible.

**Comment by Deyan2107** [comment]
> Might be [SimpleScore](https://github.com/r4g3baby/SimpleScore) but I'm not sure, can you try to remove it and check if it works again and please provide the full error/log if possible.

log file:
[latest.log](https://github.com/user-attachments/files/16041119/latest.log)

Yes, after I removed simple score the problem was fixed and allowed me to log in, but I need the plugin, should I contact their support?

**Comment by Ingrim4** [comment]
Yeah looks like something we can't fix and it will need to be addressed by the author(s) of SimpleScore.
